analyses/cell-type-ewings/exploratory_analysis/01-marker-gene-tumor-classification.Rmd

```diff
@@ -46,7 +46,7 @@ theme_set(
 repository_base <- rprojroot::find_root(rprojroot::is_git_root)
 
 # The current data directory, found within the repository base directory
-data_dir <- file.path(repository_base, "data", "current")
+data_dir <- file.path(repository_base, "data", "2024-05-01")
```

Comment:
Not spelling related, but noting that if you're going to tie this module to a specific data release then you'll want to document this required version in the module's README.

Reply:
I think we do not want to do this in general. The idea is that modules should almost always be expected to work with the current data release. In the specific case of exploratory analysis, tagging to a release might make sense, but I would rather it _output_ the release info rather than take it as hard-coded input.

Reply:
So the reason I had to do this is because I have downloaded both actual data and test data. Because I downloaded the test data second then the `current` folder symlinks to the test data. This update is _only_ in the exploratory notebooks because I needed to be able to regenerate the notebooks regardless of any other changes that have been made to symlinks. Because of that, I actually do want to tie this to a release.

Reply:
Note that this is not for the actual workflow itself. That can still take any release.

Reply:
In that case, I think a note in the exploratory notebook directory or in the notebook itself would be the way to go. I would not put this requirement in the main module readme.